docs/core/diagnostics/event-counter-perf.md

```diff
@@ -0,0 +1,606 @@
+
+## Add an action filter
+
+The sample source code is an ASP.NET Core project. You can add an [action filter](/aspnet/core/mvc/controllers/filters#action-filters) globally that will log the total request time. Create a new file named *LogRequestTimeFilterAttribute.cs*, and used the C# code below:
+
+```csharp
+using System.Diagnostics;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace DiagnosticScenarios
+{
+    public class LogRequestTimeFilterAttribute : ActionFilterAttribute
+    {
+        readonly Stopwatch _stopwatch = new Stopwatch();
+
+        public override void OnActionExecuting(ActionExecutingContext context) => _stopwatch.Start();
+
+        public override void OnActionExecuted(ActionExecutedContext context)
+        {
+            _stopwatch.Stop();
+
+            MinimalEventCounterSource.Log.Request(
+                context.HttpContext.Request.GetDisplayUrl(), _stopwatch.ElapsedMilliseconds);
+        }
+    }
+}
+```
+
+The action filter starts a <xref:System.Diagnostics.Stopwatch> as the request begins, and stops after it has completed, capturing the elapsed time. The total milliseconds is logged to the `MinimalEventCounterSource` singleton instance. In order for this filter to be applied, you need to add it to the filters collection. In the *Startup.cs* file, update the `ConfigureServices` method in include this filter.
+
+```csharp
+public void ConfigureServices(IServiceCollection services) =>
+    services.AddControllers(options => options.Filters.Add<LogRequestTimeFilterAttribute>())
+            .AddNewtonsoftJson();
+```
+
+## Monitor event counter
+
+You logged the metric to the <xref:System.Diagnostics.Tracing.EventCounter>, but unless you access the statistics from of it, it is not useful. To get the statistics, you need to enable the <xref:System.Diagnostics.Tracing.EventCounter> by creating a timer that fires as frequently as you want the events, as well as a listener to capture the events. To do that, you can use [dotnet-counters](dotnet-counters.md).
+
+Use the [dotnet-counters ps](dotnet-counters.md#dotnet-counters-ps) command, to display a list of .NET processes that can be monitored.
+
+```console
+dotnet-counters ps
+```
+
+Using the process identifier from the output of the `dotnet-counters ps` command, you can start monitoring the event counter with the following `dotnet-counters monitor` command:
+
+```console
+dotnet-counters monitor --process-id 2196 —providers Sample.EventCounter.Minimal Microsoft.AspNetCore.Hosting[total-requests,requests-per-second] System.Runtime[cpu-usage]
+```
+
+While the `dotnet-counters monitor` command is running, hold <kbd>F5</kbd> on the browser to start issues continuous requests to the `https://localhost:5001/api/values` endpoint.
```

Comment:
There was no instruction to open the browser yet.  

Given that this is an asp.net example in a non-asp.net doc set, might be a good idea to include instructions on how to deal with dev cert / say go past browser warning-- I think Rick may have an include file for that or text you can copy/paste.